source/install/install-rhel-71-server.rst

```diff
@@ -1,18 +0,0 @@
-..  _install-rhel-71-server:
-
```

Comment:
@kjkeane If you delete the page, can you make sure it redirects to the new page, e.g. here it would be:

``Please see :doc:`install-rhel-7.html`.``

Reply:
No problem, this should be updated now from last commit for both RHEL 6 and RHEL 7.